Development

heromaster opened the pull request:
- Renumbered locations to match the structure of 1.0.5 RUS version
- Renamed some locations since they were renamed in 1.0.5 RUS version too
- Reformatted and cleaned the code in 500 locations
- Restructured 096_shop
- Cleaned 005_onobjsel, it doesn't contain the dexcheats menu anymore, may be reintegrated after I've gone through the other locations
- Integrated the 1.0.5 RUS into 500 locations and kept the community edition changes
- Added the functionality to give your character a nickname (var: $nickname)

heromaster committed:
Preparation

heromaster committed:
Auto merge

heromaster committed:
First 100 locations cleaned and integrated

heromaster committed:
Bug fixes

Fixed some bugs which occured by integrating.

heromaster committed:
Bug fixes #2

heromaster committed:
Bug fixes #3

heromaster committed:
Next 100 locations

heromaster committed:
001 - 099 revisited

heromaster committed:
001 - 099 revisited

- Uniformed prices: xxxx r/RUB/rubles to x,xxx rubles
- Fixed price display errors caused by auto correcting: x, xxx rubles to
x,xxx rubles

heromaster committed:
100 - 199 revisited

heromaster committed:
100 - 199

- Uniformed prices
- Corrected errors cause by auto correction

heromaster committed:
More cleanup in 001 - 199

Regex; /(:)\ /\1/g

heromaster committed:
More cleanup in 001 - 199 #2

Simple substitution used:
RAND --> rand
END --> end
ACT --> act
LCOLOR --> lcolor
VIEW --> view
GT --> gt
<B> --> <b>
</B> --> </b>
" > --> ">
gt' --> gt '
gs' --> gs '
act' --> act '

heromaster committed:
More cleanup in 001 - 199 #3

Regex: /(,)(?!\ )/\1\ /g

heromaster committed:
More cleanup in 001 - 199 #4

Regex:
/(?<!\ |\+|\-|\/|\*|>|<)(=)(?!\ |"|=)/\ \1\ /g
/\ {2,}/\ /g

heromaster committed:
More cleanup in 001 -199 #5

Regex: /(color=)(?!")(.+?)>/\1"\2">/g

heromaster committed:
More cleanup 001-199 #6

Regex: /(<)(?!\ |<|>|=|\/|.+?>)/\ \1\ /g

Surrounded manually > chars with whitespaces

morgolon commented:
This looks awesome! I'm going to go ahead and merge each file in one at a time for a bit and see how that works. I might need some help understanding some of these changes, I'll post in here

heromaster committed:
More cleanup 001 - 199 #7

Regex: /(?<!\ |\+|\-|\/|\*|>|<)(=)(?!\ |"|=)/\ \1\ /g

heromaster commented:
Sure, go ahead. I'll try to help if I can. The first 200 locations should not be bugged although I've a problem merging them together into the TXT2GAME format. Probably has something to do with the **scale.table** location which was not extracted into a seperate file name but is included in the 005_onobjsel file. Also in the original file there are location names beginning with a $ sign. Some weren't extracted correctly either but I did not found them yet. They are probably hanging in th 400 - end files. Others were named with a capital S like Snpc or Sdiskogroup.

All other locations after that are imported into a qsp file without any problems.

On my test run I've cut out what was merged and created another qsp which I merged into the first one.

Btw: I've commented what I added myself with this code style:

```
!!!0.2.6 extension
<new code here>
!!!
```

I also made it more clear what were previous community editions by using:

```
!!!WD: <comment>
<community code
!!!
```

For now I just restructured the code base which should make it easier in further updates to add custom code. I just translated what was obvious wrong even for me as a non-native English speaker.

Currency in the first 200 locations should now be in the format 1,000 and buy actions should now be uniformed in to (1,000 rubles).

morgolon commented:
I only wrote the splitter to split on `#` locations, I think `$` locations are scripts, like menu items and such. One problem I've had was a bunch of files were not ended with a newline and that made the merge script output garbage. I fixed the merge script in one of the recent commits, you should check that out, it might fix some problems.

heromaster committed:
Notepad++ Syntax highlighting for quest

This goes into %APPDATA%/Notepad++

If you don't have any user defined languages, simple override the
existing the file
If you have already one or more then merge them together

heromaster commented:
Nope, didn't worked also with your new version. I'm currently fixing all the wrong **"** and **'**. Sometimes there are opened quotation marks but the ending were missing. Maybe that's why the import also fails. Although the parser seems really forgiving about the errors, but for importing reasons it might just get stuck.

heromaster committed:
More cleanup 001 - 199 #8

- Removed whitespace chars at end of lines
- Fixed some bugs
- Replaced ! comments with !! and end block comment ' with '! to support
the custom highlight parser for Notepad++

heromaster committed:
Fixed the name of Katja Meynold

- Fixed all varations of Katya, Kate, Katie to Katja

heromaster committed:
Fixed the name of Dimka Nosov

- Fixed all variations of Dima, Dimo to Dimka

heromaster committed:
Replaced action description

- Replaced Get_Away with Leave

heromaster committed:
Next 100 locations

heromaster committed:
More fixes in 001 - 099

- Fixed more missing changes from the 1.0.5 RUS version
- Moved 085_pharmacy to 560_pharmacy

heromaster committed:
More fixes in 001 - 099

Replaced Katia with Katja

heromaster committed:
Another round of fixes in 001 - 099

heromaster committed:
Another round of fixes 001 - 099

- Changed last name of Igor to Kruglov, it was named Nosov before
- Renamed all variables related to Artem from artom to artem, now they
are uniform, before it was sometimes a mix and match between artom...
and artem... named vars

heromaster committed:
More fixes in 100 - 199

- Renamed 169_motherQW to 561_motherQW
- Fixed some instances where not everything was updated to 1.0.5 RUS

heromaster committed:
More cleanup 200 - 299

- Deleted 204_gaptek because it was renamed to aptek and does already
exists as 069_aptek
- Moved 238_selfPorno to 563_selfPorno
- Moved 278_hairsalon to 564_hairsalon
- Moved 289_budgetclothes to 565_budgetclothes
- Cleaned and fixed more stuffs in 200 - 299 locations

heromaster committed:
Next 100 locations

heromaster committed:
Last round of fixing 001 - 299

- All missing changes from the RUS 1.0.5 version should be integrated
- Last visit before going back to recheck what was changed by the
community version and check if all changes are still in this version

heromaster committed:
Last round of fixing 300 - 399

- All missing changes from the RUS 1.0.5 version should be integrated
- Last visit before going back to recheck what was changed by the
community version and check if all changes are still in this version
- Integrated commits:
- SHA: f677bdb505a8193a85e30942677974aaa0ccaa87
- SHA: 09f50ee765c122d5c8237693120cb22cfc175060
- SHA: eb9bc0756395d4c116d36a1ef19a0491e5280b40
- SHA: 5d2e8a95cf1803786ff6a05c2ccca4e9da7577be
- centerflag982@SHA:a365577acb9239340d6385ca88ebf9313a63eda3

heromaster committed:
Integrated missing translation improvements

Integrated commits: 59a46c65c6feac3488ceb1cfdd266c781ce07474 from
@centerflag982
a365577acb9239340d6385ca88ebf9313a63eda3 from @centerflag982

heromaster committed:
More fixes and changes

- Fixed all images path from \ to /
- Toilet and bathroom events should have the same descriptions
everywhere for Enema, Vibrator & butt plug insertion/removal, Washing
off cosmetics, cleaning clothes from cum, cleaning body from cum and
cleaning face from cum
- Cleaning body on a sink now takes 15 min and requires to have at least
one clean panty
- Reduced time to remove cum from clothes from 15 to 10 mins
- Washing off cosmetics now also cleans cum from face

heromaster committed:
Fixed missing ' char in onobjsel

heromaster committed:
Next 100 locations

heromaster committed:
Last round of fixing 001 - 499

heromaster commented:
After cleaning all the invalid ' I can import the whole game now without any problems.

heromaster committed:
Last 82 locations

heromaster committed:
Fixed bugs discovered after importing into qgen

heromaster committed:
Done

Everything fixed, every location in its correct place

heromaster committed:
Transfered latest commits and fixed some bugs/translation errors

- Commits transfered:; 254948c21e7ccc627e046f5bd38a874720ab3c69 and
808f10a9d51b332b489dd046c3aa4898f0e90237

heromaster commented:
Ok, I fixed all the bugs, transfered the latest commits and tested it. Looks good. In qgen all locations are there and the game is playable :)

heromaster committed:
First bug fixed

- Somehow the wardrobe got broken, fixed
